src/NuGet.Clients/VsExtension/project.json

```diff
@@ -13,8 +14,8 @@
     "Microsoft.VisualStudio.Shell.Interop.8.0": "8.0.50727",
     "Microsoft.VisualStudio.Shell.Interop.9.0": "9.0.30729",
     "Microsoft.VisualStudio.Threading": "14.0.50702",
-    "Microsoft.VisualStudio.OLE.Interop": "7.10.6070",
-    "Microsoft.VSSDK.BuildTools": "14.0.23107"
+    "Microsoft.VSSDK.BuildTools": "14.0.23107",
```

Comment:
Added the secondary dependencies to lock down the versions.
These dependencies are licensed under the Microsoft .NET Library, Microsoft Windows Azure Libraries for .NET, and Microsoft ASP.NET Tools licenses.

Only Newtonsoft.Json is MIT License, and 6.04 is approved for VS